example-apps/token-ops/src/styles.css

```
@@ -0,0 +1,3035 @@
+:root {
+  color: #1a1f2b;
+  background: #eef0f5;
+  font-family:
+    "Inter",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)